kevm

```diff
@@ -52,13 +52,15 @@ run_kast() {
 }
 
 run_prove() {
-    local def_module run_dir additional_proof_args
+    local def_module run_dir additional_proof_args ledit
 
     def_module="$1" ; shift
 
+    command -v ledit >/dev/null 2>&1 && ledit=ledit
+
     additional_proof_args=()
     ! $debug || additional_proof_args+=(--debug)
-    ! $repl  || additional_proof_args+=(--haskell-backend-command "$k_release_dir/bin/kore-repl --repl-script $kevm_dir/kast.kscript")
+    ! $repl  || additional_proof_args+=(--haskell-backend-command "$ledit $k_release_dir/bin/kore-repl --repl-script $kevm_dir/kast.kscript")
```

Comment:
What is the purpose here?

Reply:
I made this more general in the new commit, but the motivation is to use `ledit` to enable line editing in the repl command line (like editing commands, search previous commands, etc.).